apps/condo/lang/en/messages/MESSAGE_FORWARDED_TO_SUPPORT/default.njk

```diff
@@ -0,0 +1,12 @@
+Mobile OS: {{ message.meta.os }}
```

Comment:
one more language to learn :( what is advantage here over JS function ?

Reply:
No, this is the [popular template engine](https://mozilla.github.io/nunjucks/templating.html). We are already using it for code generation (see `packages/@core.codegen`)